### ISSUE TITLE
Licence of galos_db

### ISSUE THREAD

**Issue by mattyoung101** [nixpulvis/galos#73]
Hi,

Firstly, thank you for this project! I can see you've put a lot of work into this. I have run galos_map and it's really cool.

I'm building [something similar](https://github.com/mattyoung101/edtear) to galos_db, with a focus on ingesting and storing trade data. I'm planning to make a trade calculator in the future using linear programming.

I have used some of your galos_db SQL queries, particularly the `listings` query. I was just wondering firstly if you are OK with this, and furthermore what licence you would put galos_db under, and if this licence would comply with my project's ISC licence (since I have derived from your codebase).

Thanks!

**Comment by nixpulvis** [comment]
I'd be happy to put a permissive license in here.

My philosophy around this project is essentially that it's all free. It's a passion project I touch from time to time when I play the game. It's based off the great work of the game itself, and while it has been a good amount of work, it makes me really happy to know anyone else is getting some value from it.

It would be cool if it was easier for people to contribute, but I won't make that get in the way of using anything you find here in your own projects.

**Comment by mattyoung101** [comment]
Thank you! That would be much appreciated when you have the chance :) I have mentioned you and linked to this project from my project's README as well. Galos has been very valuable to me, so I appreciate it a lot.

In terms of contributing, I was initially going to use galos_db on its own to populate the database for my trade route tool, but I got blocked by #32. I did have a go at trying to figure out what was the issue there, but unfortunately I'm not really that good at SQL so I couldn't figure it out. The only real difference is that I use a much simplified schema that only has the minimum data required to compute trade routes (so is much less useful than galos_db in general).
